modules/atomic-widgets/assets/js/editor/div-block-view.js

```diff
@@ -78,13 +80,27 @@ const DivBlockView = BaseElementView.extend( {
 
 	onRender() {
 		BaseElementView.prototype.onRender.apply( this, arguments );
+		this.handleLink();
 
 		// Defer to wait for everything to render.
 		setTimeout( () => {
 			this.droppableInitialize();
 		} );
 	},
 
+	handleLink() {
+		const linkSettings = this.model.getSetting( 'link' )?.value?.destination;
+
+		if ( 'a' !== this.tagName() || ! linkSettings ) {
```

Comment:
Maybe extract this condition to a function to increase the code readability?
WDYT?